content/docs/tasks/traffic-management/secure-ingress/index.md

```diff
@@ -452,6 +452,80 @@ only this time for host `bookinfo.com` instead of `httpbin.example.com`.
         `"""`
     {{< /text >}}
 
+## Configure end-user authentication on ingress gateway
+
+To support end-user authentication, the Istio ingress gateway sets up a JWT
+authentication policy in the `istio-ingressgateway` file. The ingress gateway
+rejects the unauthenticated requests and the request can't access the services
+inside the mesh. This section shows how to use the authentication policy to
+setup the end-user authentication for the Istio ingress gateway.
+
+1. To verify the setup, run the following curl command and confirm a return
+   value of 200:
+
+    {{< text bash >}}
+    $ curl -I -HHost:httpbin.example.com http://$INGRESS_HOST:$INGRESS_PORT/status/200
+    {{< /text >}}
+
+1. Add the policy requiring the end-user JWT authentication for the
+   `istio-ingressgateway` service.
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -f -
+    apiVersion: "authentication.istio.io/v1alpha1"
+    kind: "Policy"
+    metadata:
+      name: "ingressgateway"
+      namespace: istio-system
+    spec:
+      targets:
+      - name: istio-ingressgateway
+      origins:
+      - jwt:
+          issuer: "testing@secure.istio.io"
+          jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
+      principalBinding: USE_ORIGIN
+    EOF
+    {{< /text >}}
+
+1. Run the following curl command:
+
+    {{< text bash >}}
+    $ curl -I -HHost:httpbin.example.com http://$INGRESS_HOST:$INGRESS_PORT/status/200
+    {{< /text >}}
+
+1. Note the return value of 401. Istio returns this error code value because
+   the server expects a JWT but the ingress gateway did not provide one:
+
+    {{< text bash >}}
+    $ HTTP/1.1 401 Unauthorized
```

Comment:
Please split commands from output. I cannot tell when one ends and the other begins.

Reply:
Done.